webcompat/static/css/development/components/filter.css

```diff
@@ -61,3 +61,10 @@
   .wc-Filter--close.is-active {
     background-color:var(--wc-state-close);
   }
+  .wc-Filter--appliedlabel{
+    padding-right: 40px;
+    background-image: url(/img/invalid.svg);
```

Comment:
It's on the /issues page, there will be a new DIV wc-AppliedLabels just after wc-IssueSearch. If you click a label in the search results list (to the right of each issue), the DIV will have children. See screenshot in #795.